BUILD.md

```diff
@@ -160,21 +165,42 @@ A YDB CLI binary can be found at:
 ydb/apps/ydb/ydb
 ```
 
-### Unit tests
+## Run tests
```

Comment:
One more section is really missing: how to just run the locally built server and connect to it with CLI, without any tests.

Reply:
Let's add it later.